packages/blade-svelte/src/components/Link/BaseLink/BaseLink.svelte

```diff
@@ -0,0 +1,292 @@
+<script lang="ts">
+  import './baseLink.css';
+  import type { Snippet } from 'svelte';
+  import { cva } from 'class-variance-authority';
+  import { makeAccessible, makeAnalyticsAttribute, metaAttribute, MetaConstants } from '@razorpay/blade-core/utils';
+  import { useInteraction } from '../../../utils/useInteraction';
+  import type { ActionStatesType, ColorType } from '../../../types';
+
+  // Icon component type - placeholder for now
+  // TODO: Replace with actual Icon component when available
+  type IconComponent = any;
+
+  interface BaseLinkProps {
```

Comment:
TODO: Once we have few more components, figure out if we want to move types to a common place. Types can act as contracts to help avoid API drift and React vs Svelte. // @saurabhdaware